strawberry/channels/testing.py

```diff
@@ -144,9 +144,9 @@ async def subscribe(
             message_type = response["type"]
             if message_type == NextMessage.type:
                 payload = NextMessage(**response).payload
-                ret = ExecutionResult(payload["data"], None)
+                ret = ExecutionResult(payload.get("data"), None)
                 if "errors" in payload:
-                    ret.errors = self.process_errors(payload["errors"])
+                    ret.errors = self.process_errors(payload.get("errors") or [])
```

Comment:
> In case you want to use .get, wouldn't .get("errors", []) be shorter than .get("errors") or []

A littlebit offtopic, but there are cases where a `or` makes more sense than a default value on `get`, e.g. to get a robust `False` value but not `None`:

```
d = dict(a=1, b=None)
value = d.get('b', False)  # value is None
value = d.get('b') or False  # value is False
```

Reply:
It causes mypy error for some reason